src/main/java/com/unb/tracker/TrackerController.java

```diff
@@ -21,12 +21,23 @@ public String index(ModelMap map) {
     }
 
     @Autowired
+    private CourseRepository courseRepository;
+    @RequestMapping(value="/addCourse", method = RequestMethod.GET, produces = {MediaType.TEXT_HTML_VALUE})
+
+    public @ResponseBody
+    String addNewCourse (@RequestParam String name) {
```

Comment:
This is going to be interesting because Mac Jones and I are creating the actual course form... we're going to have some conflicts

Reply:
This was just a temporary mapping so that we could test out the course list!